mcp/pacemaker.sysconfig.m4

```diff
@@ -1,8 +1,10 @@
+ifelse(SUPPORT_CMAN, 1,
 # Turn on special handling for CMAN clusters in the init script.
 # Without this, fenced (and by inference, cman) cannot reliably be made
 # to shut down.
 # PCMK_STACK=cman
 
+)dnl
```

Comment:
Did you consider defining e.g. @SYSCONFIG_CMAN_OPTS@ using configure, instead of m4? That would avoid having to add new file handling (.in is widely used already), and it would allow hacluster/haclient to substituted correctly in the same pass. It could also restrict the definition to when the init script is in use.

On an irrelevant side note, I notice this we don't provide any equivalent functionality for running CMAN under systemd or upstart, but that would be a strange hybrid :)